WordPress/gutenberg
# Panel: Re-add aria-expanded (accidentally removed)

This update re-adds the `aria-expanded` prop to the trigger `button` within` PanelBody`. It was accidentally removed in [this update](https://github.com/WordPress/gutenberg/pull/23327). Thanks to @afercia for catching it.

For context, the accidental remove came from an attempt to use [Reakit Disclosure](https://reakit.io/docs/disclosure/). Disclosure handily adds and coordinates various `aria` attributes for us. Unfortunately, we weren't able to add it at this time. When we reverted the integration, we missed re-adding `aria-expanded` (which was previously there).

This PR resolves this issue:
https://github.com/WordPress/gutenberg/issues/24036

## How has this been tested?

* Run `npm run dev`
* Enable Mac OS Voiceover
* Expand/collapse panels in sidebar
* Ensure the voiceover announces expand/collapse

## Checklist:
- [n/a] My code follows the WordPress code style. 
- [n/a] My code has proper inline documentation. 
- [n/a] I've included developer documentation if appropriate. 
- [n/a] I've updated all React Native files affected by any refactorings/renamings in this PR.
